### PR TITLE
GV-5 Add DeepLink iOS, Android

### PR DESCRIPTION
**Pull request by f16f16f16f16**
- Add DeepLink iOS, Android

### CODE REVIEW

File: android/app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -24,6 +24,15 @@
...
             </intent-filter>
+
+            <meta-data android:name="flutter_deeplinking_enabled" android:value="true" />
+            <intent-filter android:autoVerify="true">
+                <action android:name="android.intent.action.VIEW" />
+                <category android:name="android.intent.category.DEFAULT" />
+                <category android:name="android.intent.category.BROWSABLE" />
+                <data android:scheme="http" android:host="viewer.graffity.app" />
+                <data android:scheme="https" />
+            </intent-filter>
```
Comment (f16f16f16f16) [inline android/app/src/main/AndroidManifest.xml:35]: deeplinking_enabled

File: android/build.gradle [hunk 2]
```diff
@@ -1,5 +1,5 @@
 buildscript {
-    ext.kotlin_version = '1.7.10'
+    ext.kotlin_version = '1.9.10'
```
Comment (f16f16f16f16) [inline android/build.gradle:2]: change kotlin version

File: ios/Runner/Info.plist [hunk 3]
```diff
@@ -2,6 +2,8 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
+	<key>FlutterDeepLinkingEnabled</key>
+	<true/>
```
Comment (f16f16f16f16) [inline ios/Runner/Info.plist:6]: DeepLinkingEnabled

File: ios/Runner/Runner.entitlements [hunk 4]
```diff
@@ -3,6 +3,8 @@
 <plist version="1.0">
 <dict>
 	<key>com.apple.developer.associated-domains</key>
-	<array/>
+	<array>
+		<string>applinks:viewer.graffity.app</string>
+	</array>
```
Comment (f16f16f16f16) [inline ios/Runner/Runner.entitlements:8]: add applinks

File: web/public/.well-known/assetlinks.json [hunk 5]
```diff
@@ -7,7 +7,8 @@
             "namespace": "android_app",
             "package_name": "com.graffity.ar_viewer",
             "sha256_cert_fingerprints": [
-                "17:F9:2B:BD:55:F1:A3:5B:A0:69:F1:B9:2C:31:27:F0:BD:A0:BE:E4:A4:30:00:44:F2:A7:12:09:65:FB:C5:59"
+                "17:F9:2B:BD:55:F1:A3:5B:A0:69:F1:B9:2C:31:27:F0:BD:A0:BE:E4:A4:30:00:44:F2:A7:12:09:65:FB:C5:59",
+                "84:6F:DA:77:69:0C:BD:5E:0D:A1:93:9D:C9:F9:D8:08:0A:78:E4:02:CA:A6:5E:44:45:15:87:74:E7:86:D4:CA"
```
Comment (f16f16f16f16) [inline web/public/.well-known/assetlinks.json:11]: add SHA256

**Review by bankbiz — APPROVED** [review]
LGTM

**Pull request merged**